### PR TITLE
Feature/38 documents paragraph

### PR DESCRIPTION
**Pull request by gosia-mlynarczyk**
Added Documents paragraph.

### CODE REVIEW

File: modules/localgov_campaigns_paragraphs/config/install/media.type.document.yml [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+id: document
+label: Document
+description: 'An uploaded file or document, such as a PDF.'
+source: file
+queue_thumbnail_downloads: false
+new_revision: true
+source_configuration:
+  source_field: field_media_document
+field_map:
+  name: name
```
Comment (stephen-cox) [inline modules/localgov_campaigns_paragraphs/config/install/media.type.document.yml:13]: This media definiation is already supplied by the localgov_media module (https://github.com/localgovdrupal/localgov_core/blob/master/modules/localgov_media/config/optional/media.type.document.yml) and so is not needed here.

**Review by stephen-cox — CHANGES_REQUESTED** [review]
This all looks fine apart from the inclusion of the document media type definition which is part of localgov_media and causes problems with site installs. If you can remove this file I'm happy to approve this pull request.

File: modules/localgov_campaigns_paragraphs/config/install/media.type.document.yml (hunk 1 above)
Comment (gosia-mlynarczyk) [reply modules/localgov_campaigns_paragraphs/config/install/media.type.document.yml]: Removed

**Comment by stephen-cox** [comment]
Looks fine - tests are failing due to directory issues, so not related to this.

**Review by stephen-cox — APPROVED** [review]

**Pull request merged**